INSTALL

```diff
@@ -36,6 +36,7 @@ Contents
               python-4suite-xml python-simplejson python-xml \
               gnuplot poppler-utils \
               gs-common clisp gettext libapache2-mod-wsgi unzip \
+              pkg-config \
```

Comment:
Have you tried it with latest `next` branch? The installation process has been updated.